### PR TITLE
Add 6 icons and link 4 icons

### PR DESCRIPTION
**Pull request by 1030Panda**
## Description

Fixes #(issue)

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:x: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

### Icons added
* Tieba Lite (com.huanchengfly.tieba.post)
* 7723 Games (com.upgadata.up7723)
* Mi Band WatchFace Tool (tech.pingx.watchface)
* Advanced Settings (com.byyoung.setting)
* Baidu Translate (com.baidu.baidutranslate)
* QQ Music (com.tencent.qqmusic)

### Icons linked
* Mi health (linked com.mi.health to @drawable/mi_fitness)
* WeChat (linked com.oasisfeng.nevo.agents.v1.wechat to @drawable/wechat)
* Clash You (linked yos.clash.material to @drawable/clash)
* BiliBili TV (linked com.bilibili.tv to @drawable/bilibili)

### CODE REVIEW

File: svgs/advanced_settings.svg
Comment (Chefski) [inline svgs/advanced_settings.svg:1]: Lots of rough corners in this icon. Please fix them. Also not sure if it was intended but the handle doesn't connect on the left side. The icon should also fit the 160x160 content area. It's currently horizontally short
image

File: svgs/baidu_translate.svg
Comment (Chefski) [inline svgs/baidu_translate.svg:1]: The icon goes outside the content area. Please also round the endpoints
image

File: svgs/mi_band_tool.svg
Comment (Chefski) [inline svgs/mi_band_tool.svg:1]: Some corners aren't quite round. Some shapes go outside other shapes. I marked two areas which need improvement
image

File: svgs/qqmusic.svg
Comment (Chefski) [inline svgs/qqmusic.svg:1]: Please round this corner
image

File: svgs/tieba.svg
Comment (Chefski) [inline svgs/tieba.svg:1]: Please fit the icon tot he 160x160 content area & round the corners/endpoints

File: svgs/tiebalite.svg
Comment (Chefski) [inline svgs/tiebalite.svg:1]: The corners are imperfect in this icon too. Please look at the corners in the icon and fix any other ones that need fixing. Please also fit the icon to the 160x160 content area
image

**Review by Chefski — CHANGES_REQUESTED** [review]

**Comment by 1030Panda** [comment]
Ok, I'll make the changes in time. Since it's my first time doing it, it will be rough, please forgive me.

**Pull request closed without merging**